geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -81,7 +80,7 @@ def _spectrum(
             return B.exp(-(lengthscale ** 2) / 2.0 * (s ** 2))
         elif nu > 0:
             power = -nu - self.space.dimension / 2.0
-            base = 2.0 * nu / lengthscale ** 2 + from_numpy(nu, s ** 2)
+            base = 2.0 * nu / lengthscale ** 2 + s ** 2  # from_numpy(nu, s ** 2)
```

Comment:
Why is this change needed?

Reply:
Same here, I did not get why this change is needed.